.github/workflows/ci.yml

```diff
@@ -37,8 +37,8 @@ jobs:
         - { name: Release }
         - { name: Debug }
         imgui:
-        - { version: 1.89, flags: -DIMGUI_SFML_DISABLE_OBSOLETE_FUNCTIONS=ON }
-        - { version: 1.91.5 }
+        - { version: 1.91.1 }
+        - { version: 1.91.6, flags: -DIMGUI_SFML_DISABLE_OBSOLETE_FUNCTIONS=ON }
```

Comment:
Why did you swap this around?

Reply:
Doesn't it make sense to disable obsoleted functions in the latest tested version? I don't think it makes much sense to do it in the minimum supported one, which we know is not going to trigger any new failures

Reply:
https://github.com/SFML/imgui-sfml/issues/301

Please read through this issue to get caught up on past conversions we've had on the matter. Disabling obsolete functions in the newest version will break compatibility with the oldest version.

Reply:
Right, I see. Let me revert the swap. Do I leave the latest version change or just add 1.91.6 without removing 1.91.5?

Reply:
You can still update 1.91.5 to 1.91.6. That's something I plan on periodically doing as ImGui updates to make sure nothing about ImGui breaks the ImGui-SFML build.